src/styles/settings/colors/reserved-color-scale.css

```diff
@@ -1,10 +1,14 @@
 :root {
   --ec-reserved-color-error: 0, 72%, 56%;
```

Comment:
@jaimeperezpedra  @pelocho  are this still used? the default is going to be used anywhere? From what I understood the light one is going to be the new default? or am I wrong?

Reply:
Right now the default is being still used, so we can't remove it

Reply:
I think we should rename error-hover to be error-dark and do it for every type. The hover is no longer for hover, title and subtitle uses it for text without `:hover` or `:focus`. Double check *-hover is not used in EBO and chameleon and rename them.

Reply:
You're right! In the next commit, It will be changed